python-iml-manager.spec.in

```diff
@@ -275,7 +275,6 @@ if [ -d %{_localstatedir}/lib/rpm-state/%{name}/ ]; then
     cp -rf %{_localstatedir}/lib/rpm-state/%{name}/* /var/lib/chroma/
     rm -rf %{_localstatedir}/lib/rpm-state/%{name}/
     rm -rf /var/lib/chroma/repo/iml-agent
-    chroma-config bundle delete /var/lib/chroma/repo/iml-agent/7
```

Comment:
Don't remember, does doing a bundle delete touch the fs? If it does, do we want to still do some variant of this to clear out old bundles?

Reply:
We drop those from the DB during chroma-config setup during migrations.
The chroma-config bundle delete would just drop that one bundle from db via a model delete.